### ISSUE TITLE
Should clang-omp be merged into the main llvm formula?

### ISSUE THREAD

**Issue by aw1621107** [Homebrew/legacy-homebrew#47857]
Yes, yet another thing for the llvm formula :p

According to the [OpenMP/Clang github.io page](https://clang-omp.github.io/), OpenMP support has been "promoted" to [an official LLVM project](http://openmp.llvm.org/).

**Comment by UniqMartin** [comment]
I'd say we have to keep `clang-omp` until we've managed to update the core `llvm` formula to 3.7.x (see #47824 for a recently started effort). Once done, we can retire `clang-omp` and move it to the [boneyard](https://github.com/Homebrew/homebrew-boneyard).

**Comment by gaming-hacker** [comment]
i want to see swift integrate openmp libs.

**Comment by aw1621107** [comment]
@gaming-hacker Add an OpenMP option to the Swift formula, or for Swift to support OpenMP in general?

**Comment by gaming-hacker** [comment]
@aw1621107  oh, got to be the pragmas, real tuning.   i've been hacking pragmas for years and can beat any automatic parallelizer.   a compiled python-ish language on steroids

**Comment by aw1621107** [comment]
@gaming-hacker Could you elaborate? I'm not quite understanding what you're getting at

**Comment by UniqMartin** [comment]
Closing as there's currently nothing that can/needs to be done. Once our `llvm` formula is ready (read: updated) we can revisit what should happen with `clang-omp`.
